### PR TITLE
[Templates] Add 'article type' field into article listings

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This PR adds the "article type" field into article listings pages, which are generated during the build by cycling through certain types of nodes.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/15607

## Testing done
Verified the field renders via local preview

## Screenshots
See the highlighted text

image

## Acceptance criteria
- [ ] Article type is visible

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/plugins/download-drupal-assets.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ async function downloadFile(
 
 function downloadDrupalAssets(options) {
   const client = getDrupalClient(options);
+  return () => {};
```
Comment (ncksllvn) [inline src/site/stages/build/plugins/download-drupal-assets.js:77]: This was an accident...reverting this file.

File: src/site/stages/build/plugins/create-resources-and-support-section.js [hunk 2]
```diff
@@ -25,15 +25,19 @@ const BREADCRUMB_BASE_PATH = [
...
-  ENTITY_BUNDLES.SUPPORT_RESOURCES_DETAIL_PAGE,
-]);
+const articleTypesByEntityBundle = {
+  [ENTITY_BUNDLES.Q_A]: 'Question and answer',
+  [ENTITY_BUNDLES.CHECKLIST]: 'Checklist',
+  [ENTITY_BUNDLES.MEDIA_LIST_IMAGES]: 'Media list',
+  [ENTITY_BUNDLES.MEDIA_LIST_VIDEOS]: 'Media list',
+  [ENTITY_BUNDLES.SUPPORT_RESOURCES_DETAIL_PAGE]: 'About',
+  [ENTITY_BUNDLES.FAQ_MULTIPLE_Q_A]: 'FAQs',
+  [ENTITY_BUNDLES.STEP_BY_STEP]: 'Step by step',
```
Comment (kelsonic) [inline src/site/stages/build/plugins/create-resources-and-support-section.js:35]: I think this needs to be `Step-by-step` https://github.com/department-of-veterans-affairs/va.gov-team/issues/15306

**Review by kelsonic — APPROVED** [review]
LGTM! Left 1 minor comment 🙂

File: src/site/stages/build/plugins/create-resources-and-support-section.js (hunk 2 above)
Comment (ncksllvn) [reply src/site/stages/build/plugins/create-resources-and-support-section.js:35]: 🤦 thanks!

Comment (kelsonic) [reply src/site/stages/build/plugins/create-resources-and-support-section.js:35]: of course! 🙂

**Review by U-DON — APPROVED** [review]

**Pull request merged**